### PR TITLE
Get user courses

### PR DESCRIPTION
**Pull request by LeaHiHi**
Added the api and instructions to use it in GetCourses.java.

### CODE REVIEW

File: app/src/main/java/com/instructure/template/projectCodeHere/javafiedExamples/ExampleDestinationFragmentJavified.java [hunk 1]
```diff
@@ -10,7 +10,7 @@
 import com.instructure.template.R;
 
 public class ExampleDestinationFragmentJavified extends Fragment {
-
+
```
Comment (HeyBanditoz) [inline app/src/main/java/com/instructure/template/projectCodeHere/javafiedExamples/ExampleDestinationFragmentJavified.java]: Extra indent.

File: app/src/main/java/com/instructure/template/projectCodeHere/GetCourses.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            @Header("User-Agent") String userAgent
+    );
+
+    public static final Retrofit retrofit = new Retrofit.Builder()
+            .baseUrl(ApiPrefs.getFullDomain())
+            .addConverterFactory(GsonConverterFactory.create())
+            .build();
+}
+    //This is how one would implement this interface (returns json data)
+    /*GetCourses getCourses = GetCourses.retrofit.create(GetCourses.class);
```
Comment (HeyBanditoz) [inline app/src/main/java/com/instructure/template/projectCodeHere/GetCourses.java:28]: Why is this stuff commented? You could probably throw a [UnsupportedOperationException](https://docs.oracle.com/javase/7/docs/api/java/lang/UnsupportedOperationException.html) instead.

**Review by HeyBanditoz — CHANGES_REQUESTED** [review]

**Review by HeyBanditoz — CHANGES_REQUESTED** [review]
You also probably should squash your commits into one.

File: app/src/main/java/com/instructure/template/projectCodeHere/GetCourses.java (hunk 2 above)
Comment (LeaHiHi) [reply app/src/main/java/com/instructure/template/projectCodeHere/GetCourses.java:28]: because you are in an interface. you cant run it there

**Review by HeyBanditoz — APPROVED** [review]

**Pull request merged**